doc/api/esm.md

```diff
@@ -444,9 +444,10 @@ versions of Node.js (and other runtimes). New packages could be published
 containing only ES module sources, and would be compatible only with ES
 module-supporting runtimes.
 
-It is strongly discouraged to publish packages that use conditional exports
-to provide both transpiled and untranspiled versions of the same package, e.g.
-with:
+It is strongly encouraged to only publish CommonJS or use a wrapper approach
```

Comment:
i'm still not on board with "strongly encouraged"; we don't have universal agreement there.

I think that it's better to frame it as something like "If you choose this approach, please be aware that anything that depends on object identity (`===`, `instanceof`, `Map`/`Set` membership, etc) of values that are created in more than one file can cause unexpected bugs"

Reply:
"for once" is needlessly hostile; I wrote text in my suggestion.

Reply:
Your review and feedback into the process is highly valued, it means a lot getting prompt feedback as well. But for this disagreement specifically, finding the right wording is your problem to solve - for this once exactly! 

I really do mean what I said in that I do not see it as my responsibility to find consensus here on the exact wording of a disagreement you have with other members of the group. And I do not think you should be making that disagreement a blocking concern of this PR. Significant effort has been put into these docs in a way to ensure we can cut the right middle-ground. 

There is a suggest feature here - and others can disagree on that if they like. But I will not be your pen.

Reply:
The suggest feature does not work yet on multiple lines, so because of the hard word wrapping, it's highly unergonomic to make concrete suggestions at this time.

Nothing on any PR is "blocking" unless it's got a red X on it - do you see one here?

Reply:
Even if you can propose text in-line I can commit it if there is agreement, the issue is that the back-and-fourth is not practical for me to push a commit each time for this process, and while I value the privilege of being able to work on this stuff I do have limited time available for it too.

> Nothing on any PR is "blocking" unless it's got a red X on it - do you see one here?

Good to know. While docs can easily be treated as an afterthought, a lot of effort was made for this PR to try give the right advice. It's just sad to see that backfiring in bringing down the discussion, when this is a high-stakes feature with wide-ranging ecosystem implications, that could do with as much careful thought as possible.

Reply:
I disagree with this change. I think we should go back to “strongly discouraged.” It _is_ explicitly discouraged, we’re not implicitly discouraging it by encouraging something else.

I think we should revert to the original text and @ljharb can propose alternate text if he likes.

As an opinion in the docs, this statement represents an official position of Node, which in this case means the TSC. It’s irrelevant whether @ljharb agrees with it or not, only that it reflects the opinion of the TSC; and I think the “strongly discouraged” text does that. Even if the TSC can be persuaded to permit the hazard, I think it’s almost certain that they want to discourage it, and so that’s what the docs should say.

Reply:
I've gone ahead and removed all docs changes not directly related to the direct API docs for this PR. Rather, we can come back to this docs update in a future follow-up docs PR.

Reply:
@GeoffreyBooth ... I don't have strong opinions on this particular issue, but I would clarify one thing: @ljharb is a collaborator in the Node.js project and their opinion absolutely matters and is not irrelevant. We follow a consensus seeking model where consensus is built among collaborators and the TSC exists to provide a backstop in case of conflict. So the characterization in your sentence, "As an opinion in the docs, this statement represents an official position of Node, which in this case means the TSC" is incorrect... the official position of Node.js is the consensus established among the collaborators in discussions such as these.

Reply:
@jasnell I was referring to the future, as in, when these docs get published, they will reflect an official position of Node. Obviously they don’t yet reflect that, because there isn’t yet a consensus.

There’s a separate process in https://github.com/nodejs/modules/issues/402 for setting this question and reaching a decision, even if it means intervention by the TSC.

Reply:
@jasnell for what it's worth @ljharb is not actually a core collaborator of the Node.js project but given their continuing involvement over the last few years we might want to change that soon :]

(they are a tc39 member and a moderation team member on the other hand as well as a slack admin)